src/status/wallet.nim

```diff
@@ -50,6 +71,36 @@ proc initEvents*(self: WalletModel) =
 proc delete*(self: WalletModel) =
   discard
 
+proc trackPendingTransaction*(self: WalletModel, address: string, trxHash: string, trxType: PendingTransactionType, data: string) =
+  let latestBlock = getBlockByNumber("latest").parseJson()["result"].getInt
+
+  if not self.pendingTransactions.hasKey(address):
+    self.pendingTransactions[address] = @[]
+
+  self.pendingTransactions[address].add PendingTransaction(
```

Comment:
Maybe it's a GitHub bug,  but I see a space between `add` and `PendingTransaction`, that doesn't feel right

Reply:
It's an `add()` without `()`. Does look odd.